### PR TITLE
Fix missing commas

### PR DESCRIPTION
**Pull request by sethvargo**
/cc @yegle 

**Release Note**

```release-note
NONE
```

### CODE REVIEW

File: terraform/alerting/verification-server/alerts.tf [hunk 1]
```diff
@@ -268,11 +273,11 @@ EOT
   }
 
   notification_channels = [
-    google_monitoring_notification_channel.email.id
+    google_monitoring_notification_channel.email.id,
   ]
+
   depends_on = [
-    null_resource.manual-step-to-enable-workspace
+    null_resource.manual-step-to-enable-workspace,
```
Comment (sethvargo) [inline terraform/alerting/verification-server/alerts.tf:280]: This one in particular is actual failing the run

**Comment by chaodaiG** [comment]
Curious why it was not caught by the smoke test this morning https://prow.k8s.io/view/gcs/oss-prow/logs/ci-en-verification-server-terraform-smoke/1325810403162198016

**Comment by sethvargo** [comment]
@chaodaiG because everything is pinned to the old release branch.

**Pull request merged**